Edit of development section docs (#13247)
PrefectHQ/prefect: docs/recipes/recipes.md

## Contributing recipes

We're always looking for new recipe contributions! See the [Prefect Recipes](https://github.com/PrefectHQ/prefect-recipes#contributing--swag-) repository for details about how to add your Prefect recipe, share best practices with fellow Prefect users, and earn some swag.

[Prefect recipes](https://github.com/PrefectHQ/prefect-recipes) provide a vital cookbook where users can find helpful code examples and common steps for specific Prefect use cases.

> We love recipes from anyone who has example code that another Prefect user can benefit from (for example, a Prefect flow that loads data into Snowflake).

Reviewer: ```suggestion
Contribute recipes that other Prefect users could benefit from (for example, a Prefect flow that loads data into Snowflake).
```

- Code and more: Sometimes a use case is best represented in Python code or shell scripts. Sometimes a configuration file is the most important artifact &mdash; think of a Dockerfile or Terraform file for configuring infrastructure.
- All-inclusive: Share as much code as you can. Even boilerplate code like Dockerfiles or Terraform or Helm files are useful. Just *don’t share company secrets or IP*.
- Specific: Don't worry about generalizing your code. Other users will extrapolate their own unique solutions from your example.

## Tips for a good recipe README

A thoughtful README can take a recipe from good to great. Here are some best practices for a great recipe README:

- Provide a brief explanation of what your recipe demonstrates. This helps users determine quickly whether the recipe is relevant to their needs or answers their questions.
- List which files are included and what each is meant to do. Each explanation can contain only a few words.

> - Describe any dependencies and prerequisites (in addition to any dependencies you include in a requirements file). This includes both libraries or modules and any services your recipe depends on.

Reviewer: ```suggestion
- Describe any dependencies and prerequisites (in addition to any dependencies you include in a requirements file). This includes both libraries or modules and any services your recipe depends upon.
```
